src/test/java/com/connexta/search/index/IndexTests.java

```diff
@@ -187,9 +193,41 @@ void testInvalidAcceptVersion() throws Exception {
   }
 
   @Test
-  @Disabled("TODO")
-  public void testCantReadAttachment() {
-    // TODO verify 400
+  public void testCantReadAttachment() throws IOException {
+    MultipartFile fileMock = mock(MultipartFile.class);
+    doThrow(IOException.class).when(fileMock).getInputStream();
+    doReturn(MediaType.APPLICATION_OCTET_STREAM.toString()).when(fileMock).getContentType();
+    Assertions.assertThrows(
+        IndexException.class,
+        () -> indexController.index(indexApiVersion, "00067360b70e4acfab561fe593ad3f7a", fileMock));
+  }
+
+  @Test
+  public void testCannotReadAttachment() throws Exception {
+
+    MockMultipartFile fileSpy =
+        spy(
+            new MockMultipartFile(
+                "file",
+                "this originalFilename is ignored",
+                "application/json",
+                IOUtils.toInputStream(
+                    "{\"ext.extracted.text\" : \"All the color had been leached from Winterfell until only grey and white remained\"}",
+                    StandardCharsets.UTF_8)));
+    doThrow(new IOException("Bad stream")).when(fileSpy).getInputStream();
```

Comment:
❓ Can you extend MockMultipartFile instead of using a spy? I though spys were only needed for static methods, but getInputStream() is not static.

Reply:
I hadn't thought of that. I'm sure I could.